tests/api/v1/generator/base_case.py

```diff
@@ -91,6 +91,7 @@ def _template_data(self):
         return {
             'user': os.environ.get('USER'),
             'workingDirectory': os.environ['PTERO_WORKFLOW_TEST_TEMP'],
+            'environment': json.dumps(dict(os.environ)),
```

Comment:
I think that it might only be necessary to set the PATH for this to work.  It might be worth considering, I'm not sure.